src/modules/shared/infrastructure/cache/avvcatalogue.cache.ts

```diff
@@ -0,0 +1,26 @@
+import { AVVCatalogueCacheData } from '../../../orders/infrastructure/repositories/avvcatalogue.repository';
+import { AVVCatalog } from '../../../orders/legacy/model/avvcatalog.entity';
```

Comment:
Please no dependencies to legacy modeule

Reply:
refactored in #97